src/modules/mints/MintPage.tsx

```diff
@@ -1,8 +1,9 @@
-import MintChart from "../components/charts/mint-chart";
-import Layout from "../components/layouts";
+import MintChart from "src/components/charts/mint-chart";
+import Layout from "src/components/layouts";
 import { useState } from "react";
+import { NextPage } from "next";
 
-export default function Mints() {
+export const MintPage: NextPage = () => {
 	const [view, setView] = useState("volume");
```

Comment:
Is this value will have other than volume ?

Reply:
Yes, it will. I guess the useState needs to be added to its own type.